app/Http/Controllers/Auth/OAuth2Controller.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace Pterodactyl\Http\Controllers\Auth;
+
+use OAuth2;
```

Comment:
What package is giving you this global `OAuth2` namespace like this? @lancepioch is this what you were mentioning?